acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/bft/BftMiningAcceptanceTest.java

```diff
@@ -123,4 +123,52 @@ public void shouldStillMineWhenANonProposerNodeFailsAndHasSufficientValidators()
 
     cluster.verifyOnActiveNodes(receiver.balanceEquals(80));
   }
+
+  @Test
+  public void shouldMineWithIgnoringANodeInCRL() throws Exception {
+    final BesuNode minerNode1 = nodeFactory.createNode(besu, "miner1");
+    final BesuNode minerNode2 = nodeFactory.createNode(besu, "miner2");
+    final BesuNode minerNode3 = nodeFactory.createNode(besu, "miner3");
+    final BesuNode minerNode4 = nodeFactory.createNode(besu, "miner4");
+    final BesuNode minerNode5 = nodeFactory.createNode(besu, "miner5");
+    final BesuNode minerNode6 = nodeFactory.createNode(besu, "miner6");
+    try {
+      cluster.start(minerNode1, minerNode2, minerNode3, minerNode4);
+
+      cluster.startNode(minerNode5);
+      cluster.startNode(minerNode6);
+
+      cluster.verify(blockchain.reachesHeight(minerNode1, 1, 85));
+
+      final Account sender = accounts.createAccount("account1");
+      final Account receiver = accounts.createAccount("account2");
+
+      minerNode1.execute(accountTransactions.createTransfer(sender, 50));
+      cluster.verify(sender.balanceEquals(50));
+
+      minerNode2.execute(accountTransactions.createIncrementalTransfers(sender, receiver, 1));
+      cluster.verify(receiver.balanceEquals(1));
+
+      minerNode3.execute(accountTransactions.createIncrementalTransfers(sender, receiver, 2));
+      cluster.verify(receiver.balanceEquals(3));
+
+      minerNode4.execute(accountTransactions.createIncrementalTransfers(sender, receiver, 3));
+      cluster.verify(receiver.balanceEquals(6));
+
+      if (minerNode1.getTLSConfiguration().isEmpty()) {
```

Comment:
I'm a little bit confused by this if statement here. What is the reason for this?

Reply:
if you are referring to the if statement, that is needed as we are using the parameterized tests with multiple config settings the CRL config is present only for the TLS related parameterized configs and hence those nodes are rejected during connection attempts.